### PR TITLE
Handle command retry in register_addons_cmd

### PR DESCRIPTION
**Pull request by jlausuch**
The code was hardcoding the number of retries to 1, but it would
be more meaningful that that parameter is given by the test.

Failure: https://openqa.suse.de/tests/8254899#step/suseconnect_scc/50
VR: https://openqa.suse.de/tests/8255243

### CODE REVIEW

**Review by mloviska — APPROVED** [review]

**Review by pdostal — APPROVED** [review]
@asmorodskyi we use `add_suseconnect_product` also in `tests/publiccloud/` do you think that we should add this `retry` parameter there as well?

**Review by jknphy — APPROVED** [review]

**Comment by asmorodskyi** [comment]
> @asmorodskyi we use `add_suseconnect_product` also in `tests/publiccloud/` do you think that we should add this `retry` parameter there as well?

the changes done directly into the function so my understanding it will already increase amount of attempts to 3 (default value) , right ? so let's see how it will affect us and then discuss if we need to tune it up or not

File: tests/console/suseconnect_scc.pm [hunk 1]
```diff
@@ -41,7 +41,7 @@ sub run {
     assert_script_run q[SUSEConnect --list-extensions | grep -e '\(Activated\)'] if is_sle;
 
     # add modules
-    register_addons_cmd if $scc_addons;
+    register_addons_cmd($scc_addons, 5) if $scc_addons;
```
Comment (asmorodskyi) [inline tests/console/suseconnect_scc.pm:44]: there is also retry in line 37 , maybe we can extract amount of retries to a variable ?

**Review by asmorodskyi — APPROVED** [review]
LGTM , small suggestion is not a must feel free to ignore it

File: tests/console/suseconnect_scc.pm (hunk 1 above)
Comment (jlausuch) [reply tests/console/suseconnect_scc.pm:44]: yes, good point

**Comment by jlausuch** [comment]
VR https://openqa.suse.de/tests/8255471

**Pull request merged**